packages/nimble-components/src/table/components/row/tests/table-row.spec.ts

```diff
@@ -221,6 +223,46 @@ describe('TableRow', () => {
 
             expect(listener.spy).not.toHaveBeenCalled();
         });
+
+        it('shows expand-collapse button when rowHierarchyLevel is "topLevelParent"', async () => {
+            const pageObject = new TableRowPageObject(element);
+            await connect();
+            element.rowHierarchyLevel = TableRowHierarchyLevel.topLevelParent;
+            await waitForUpdatesAsync();
+
+            expect(pageObject.getExpandCollapseButton()).toBeDefined();
+        });
+
+        it('hides expand-collapse button when rowHierarchyLevel is "parent"', async () => {
```

Comment:
Should add test cases for `rowHierarchyLevel`s `undefined` and `leaf`.

Reply:
Not after refactor.